weave-js/src/assets/icons/icon-not-visible.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
i think we wand icons with viewbox 0 0 24 24, i think depending on the way you export it from figma it can be so